README.md

```diff
@@ -14,4 +14,25 @@ The repository is using a monorepo strategy in order to separate constraints int
 - [`@logossim/core`](/packages/@logossim/core/README.md)
 - [`@logossim/components`](/packages/@logossim/components/README.md)
 - [`@logossim/page`](/packages/@logossim/page/README.md)
-- [`@logossim/component-creator`](/packages/@logossim/component-creator/README.md)
\ No newline at end of file
+- [`@logossim/component-creator`](/packages/@logossim/component-creator/README.md)
+
+# Installation
+
+Requirements
+
+- [NodeJS](https://nodejs.org/en/)
+- [Yarn](https://yarnpkg.com/)
```

Comment:
Can you please change the URL to `https://classic.yarnpkg.com/lang/en/`, just to clarify it's Yarn v1 (classic) we're talking about?

Reply:
Done!